### DISCUSSION TITLE
Using Microsoft Search instead if sharePoint search will result in minimal cards + duplicate results

### DISCUSSION THREAD

**Discussion by mvcsharepointdev** [microsoft-search/pnp-modern-search#2864] [Q&A]
I have the following settings inside my Search Box:-
 
![image](https://user-images.githubusercontent.com/79726511/226188333-f6cb64c9-a335-41c6-aeaf-f2a5652eb2b0.png)

![image](https://user-images.githubusercontent.com/79726511/226188342-b5e33600-516f-425a-a6f3-0f0172a5fffc.png)

![image](https://user-images.githubusercontent.com/79726511/226188353-11980520-b5bf-460b-8926-b39580cdbfc9.png)

And the following settings inside the Search Result:-

![image](https://user-images.githubusercontent.com/79726511/226188415-ef44345b-d679-4656-9939-3107609d360a.png)
![image](https://user-images.githubusercontent.com/79726511/226188426-c7c3629c-4448-4c39-bfe1-b48f80329db7.png)
![image](https://user-images.githubusercontent.com/79726511/226188438-622d683e-dcb1-4386-aecf-3f57d36e6da6.png)

![image](https://user-images.githubusercontent.com/79726511/226188446-785c31a5-4029-4ea6-a54f-954e26d4a53d.png)

and the following settings inside the filter:-

![image](https://user-images.githubusercontent.com/79726511/226188457-5d96d901-a55c-4158-8986-17943f69c343.png)

now if i am using SharePoint search everything will work fine. but if i use Microsoft search i will get those 2 main issues:-

1) if i search for the work "scope of work" for example, i will get 3 results (2 are duplicates), as follow (if i use SharePoint search i will only get 2 unique results): -

![image](https://user-images.githubusercontent.com/79726511/226188678-c2f62b5f-862f-4abf-a266-4380d8d90b01.png)

2) the default card for the SharePoint search will show these info:-

![image](https://user-images.githubusercontent.com/79726511/226188764-0171bf0f-62c4-47d3-9a46-38f37c58ad61.png)

while when using Microsoft search the info will be minimal:-

![image](https://user-images.githubusercontent.com/79726511/226188804-a4a196fa-baec-473d-b4a7-2ada1432be96.png)

so how i can fix the 2 issues, so the Microsoft search will work similar to the SharePoint Search ??

**Comment by wobba** [comment]
You should verify your Graph queries using the Graph Explorer. If you get duplicates that is an API issue outside the web parts control.

Also, if you use the debug template you see what data is returned by default. Then you can add more fields as needed to ensure you have all the data you need via the slots.

We understand this is a bit tricky and require a lot of upfront knowledge to get working 100% and we may update the scenario docs more in the future to help resolve some of the setup.

**Comment by cwencv** [comment]
@wobba Thank you for the explanation. We have a similar issue but using the Microsoft Search data source. 
How can I analyse the API request sent from pnp search to analyse the cause of this issue.

We also have a lot of duplicates in the search result.
Thank you in advance.

**Reply by kasperbolarsen** [reply]
Did you try your query in the MS Graph Explorer?
